tests/ml_tools.py

```diff
@@ -0,0 +1,117 @@
+import awkward as ak
+import dask_awkward as dak
+import numpy as np
+
+import coffea.ml_tools
+
+
+def prepare_jets_array():
+    # Creating jagged Jet-with-constituent array
+    NJETS = 2000
+    NFEAT = 100
+    jets = ak.zip(
+        {
+            "pt": ak.from_numpy(np.random.random(size=NJETS)),
+            "eta": ak.from_numpy(np.random.random(size=NJETS)),
+            "phi": ak.from_numpy(np.random.random(size=NJETS)),
+            "ncands": ak.from_numpy(np.random.randint(1, 50, size=NJETS)),
+        },
+        with_name="LorentzVector",
+    )
+    pfcands = ak.zip(
+        {
+            "pt": ak.from_regular(np.random.random(size=(NJETS, NFEAT))),
+            "eta": ak.from_regular(np.random.random(size=(NJETS, NFEAT))),
+            "phi": ak.from_regular(np.random.random(size=(NJETS, NFEAT))),
+            "feat1": ak.from_regular(np.random.random(size=(NJETS, NFEAT))),
+            "feat2": ak.from_regular(np.random.random(size=(NJETS, NFEAT))),
+        },
+        with_name="LorentzVector",
+    )
+
+    idx = ak.local_index(pfcands.pt, axis=-1)
+    pfcands = pfcands[idx < jets.ncands]
+    jets["pfcands"] = pfcands[:]
+
+    ak_jets = jets[:]
+    ak.to_parquet(jets, "ml_tools.parquet")
+    dak_jets = dak.from_parquet("ml_tools.parquet")
+    return ak_jets, dak_jets
+
+
+def triton_testing():
```

Comment:
unless `__name__ == "__main__"` is picked up by pytest you'll eventually need to turn this into something named `test_triton` or similar for the CI to pick it up. 

If I missed something about how to hook it in let me know!